### ISSUE TITLE
Mailing list?

### ISSUE THREAD

**Issue by jamercee** [unbit/uwsgi#2230]
Is the mailing list still alive? If so, could someone point me to where it might be?  I've tried http://lists.unbit.it/cgi-bin/mailman/listinfo/uwsgi, but the site has been down for quite some time.

**Comment by rdeioris** [comment]
@jamercee nope, it has been deactivated months ago. You can open issues here or on stackoverflow. We are evaluating a google group btw.

**Comment by jamercee** [comment]
I thought so...just wanted to be sure. Thanks

**Comment by jamercee** [comment]
I thought so...just wanted to be sure. Thanks
